src/txn/txn.c

```diff
@@ -1630,6 +1483,12 @@ __wt_txn_commit(WT_SESSION_IMPL *session, const char *cfg[])
 
     /* Set the commit and the durable timestamps. */
     WT_ERR(__wt_txn_set_timestamp(session, cfg));
+    WT_ASSERT(session,
+      !F_ISSET(txn, WT_TXN_HAS_TS_COMMIT) || txn->commit_timestamp <= txn->durable_timestamp);
+
+    if (F_ISSET(txn, WT_TXN_HAS_TS_COMMIT) && txn->commit_timestamp > txn->durable_timestamp)
```

Comment:
I think this check is already performed as part of `__wt_txn_set_timestamp`?

Reply:
I thought so too -- I was looking for that... :) 

I think the only place we check is `__wt_txn_set_durable_timestamp()`, which is the appropriate error message. So, shall I just remove this check entirely?